bitstamp/client.py

```diff
@@ -57,12 +57,11 @@ def _construct_url(self, url, base, quote):
         """
         Adds the orderbook to the url if base and quote are specified.
         """
-        #This condition allows for both None and False values
-        if not base and not quote:
+        if (base is None) and (quote is None):
             return url
         else:
-            #The join method is quicker than standard concatenation
-            url.join([base.lower(), quote.lower(), "/"])
+            #TODO: The join method is quicker than standard concatenation
```

Comment:
As far as I'm concerned, it is fine to do it with the +. I will remove the TODO.